builder-base/Makefile

```diff
@@ -81,7 +83,7 @@ images-minimal: FINAL_STAGE_BASE=minimal-copy-stage
 images-minimal: VARIANT=minimal
 
 .PHONY: images-%
-images-%: buildkit-check download-golang
+images-%:
```

Comment:
did u put the buildkit-check call in the prow job yaml?

Reply:
yeah these are running as a pre-req in the prow jobs https://github.com/aws/eks-distro-prow-jobs/blob/main/jobs/aws/eks-distro-build-tooling/builder-base-postsubmits.yaml#L62